hls4ml/templates/oneapi/firmware/nnet_utils/nnet_recurrent.h

```diff
@@ -315,6 +358,192 @@ void simple_rnn(const data_T &data, res_T &res, const typename CONFIG_T::weight_
     }
 }
 
+//----------------------
+// SimpleRNN with pytorch biases
+//----------------------
+
+struct simpleRNN_pytorch_config {
```

Comment:
A minor annoyance is that this breaks naming convention, I was expecting `simple_rnn_...`

Reply:
Actually, this is never used (and similarly also not for quartus). It seems like the `simpleRNN_config` is used for all cases. Of course that one also violates the naming convention.

Reply:
My suggestion is that we remove `simpleRNN_pytorch_config`  and rename `simpleRNN_config`, though we could keep the two if we really wanted to. That would complicate the templates a touch for no real gain. (I truthfully was never much of a fan of the base config templates that get overriden.)

Reply:
https://github.com/JanFSchulte/hls4ml/pull/9 does the simple approach to rename `simpleRNN_config` as `simple_rnn_config` and remove `simpleRNN_pytorch_config` altogether, if you agree with that approach.